docxcompose/properties.py

```diff
@@ -13,50 +68,144 @@ class CustomProperties(object):
     """
     def __init__(self, doc):
         self.doc = doc
+        self.part = None
         self._element = None
 
         try:
             part = doc.part.package.part_related_by(RT.CUSTOM_PROPERTIES)
         except KeyError:
-            pass
+            self._element = parse_xml(self._part_template())
         else:
+            self.part = part
             self._element = parse_xml(part.blob)
 
-    def dict(self):
-        """Returns a dict with all custom doc properties"""
-        if self._element is None:
-            return dict()
+    def _part_template(self):
+        template_path = os.path.join(
```

Comment:
shouldn't you use `setuptools` `pkg_resources` here?

Reply:
Amended.